.happy/terraform/modules/sfn_config/nextstrain.wdl

```diff
@@ -43,6 +43,9 @@ task nextstrain_workflow {
     set -Eeuxo pipefail
     shopt -s inherit_errexit
 
+    df
```

Comment:
gotta add `1>&2` so miniwdl doesn't eat stdout!

Reply:
damn it you're right.

Reply:
ok fixed.